cmd/hub/mcscontroller/main.go

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 // The controller is responsible for coordinating a common view of exports
 // and imports on a shared (a.k.a. hub) cluster, and replaces the, per-workload-
 // cluster mapping, currently provided by Submariner's lighthouse component.
-package mcscontroller
+package main
```

Comment:
shall the parent folder be called  `mcshub` as well as in the pkg folder?

Reply:
or maybe put the main under `pkg/mcshub` like the agent's structure, for coherency?

Reply:
can't have main and the controller test files in the same directory
(unable to import package main which is required for executable)
Renamed from cmd/hub.mcscontroller to cmd/mcshub

Reply:
in the agent they put the controller under controller subdir ..